tests/executables_src/testRebotConnectionTimeouts.cpp

```diff
@@ -56,8 +56,10 @@ BOOST_FIXTURE_TEST_CASE(testOpenConnection, F) {
 BOOST_FIXTURE_TEST_CASE(testReadTimeout, F) {
   uint32_t timeout_sec = 1;
   auto accetable_completion_time = std::chrono::seconds(timeout_sec * 5);
-  Device d("sdm://./rebot=localhost," + std::to_string(rebotServer.port()) + ",mtcadummy_rebot.map," +
-      std::to_string(timeout_sec));
+  Device d("(rebot?ip=localhost&port=" + std::to_string(rebotServer.port()) + "&map=mtcadummy_rebot.map&timeout=" +
+      std::to_string(timeout_sec)+")");
+
+  //Device d("rebot?ip=localhost&port=5001&map=mtcadummy_rebot.map");
```

Comment:
Coding style: Don't leave commented code lines.

Reply:
Why I do not see them in Jenkins branch :(

Reply:
This is not part of the unit test and automatic code formatting. The formatter cannot  know if it is a real comment or a  line of code which has been commented out during development.
It is the same category as "remove debug cout". It's valid C++, so the automatic tools cannot find this.